packages/plugins/geth/src/blockchainProcessLauncher.js

```diff
@@ -15,14 +15,15 @@ export class BlockchainProcessLauncher {
     this.isDev = options.isDev;
     this.client = options.client;
     this.embark = options.embark;
+    this.isWhisper = options.isWhisper;
```

Comment:
I think, in `blockchain.js`, we differentiate whisper node and blockchain node with the presence of the comms config. Not saying it's perfect, but we could do it the same way if you wanted

Reply:
Good idea, will update to use this.